db/migrate/20200325033438_add_extra_fields_to_user.rb

```diff
@@ -5,8 +5,8 @@ def change
     add_column :users, :username, :string
     add_column :users, :firstname, :string
     add_column :users, :lastname, :string
-    add_column :users, :contact, :bigint
-    add_column :users, :country, :string
-    add_column :users, :zipcode, :integer
+    add_column :users, :contact, :string
```

Comment:
always create new migration. please do not change like this. now all have to rollback this migration then it will work